run

```diff
@@ -1,6 +1,27 @@
 #!/bin/bash
 set -euo pipefail
 
+log() {
+    msg="$1"
+
+    logline="[$(pwd) $0] $msg\n"
+
+    printf "$logline" >&1
+    printf "$logline" >&2
+}
+
+if [ -n "${HAPROXY_SYSLOGD-}" ]; then
+    log "starting syslogd with container stdout"
+
+    # A hack to redirect syslog to stdout.
+    # This is necessary because haproxy only logs to syslog.
+    ln -sf /proc/$$/fd/1 /var/log/container_stdout
+
+    # Start syslog
+    /usr/sbin/syslogd -f "/marathon-lb/syslogd/syslog.conf"
```

Comment:
.. and if it crashes (or is killed due to an OOM event) who restarts it? if you're going to run multiple daemons in a single container then you need a process manager (one w/ built-in support for dealing with zombies). anything else is probably not production-grade